components/kyma-environment-broker/internal/broker/instance_update.go

```diff
@@ -107,12 +107,16 @@ func (b *UpdateEndpoint) Update(_ context.Context, instanceID string, details do
 	}
 
 	if b.processingEnabled {
-		instance, err := b.processContext(instance, details, lastProvisioningOperation, logger)
+		instance, suspendStatusChange, err := b.processContext(instance, details, lastProvisioningOperation, logger)
 		if err != nil {
 			return domain.UpdateServiceSpec{}, err
 		}
 
-		return b.processUpdateParameters(instance, details, lastProvisioningOperation, asyncAllowed, logger)
+		// NOTE: KEB currently can't process update parameters in one call along with context update
```

Comment:
Just thinking, maybe we should never execute processUpdateParameters if any context attribute was changed?

Reply:
afaict, the code ignores any other context arguments, it processes only `Active` flag in the broker endpoint.